server/gokbg3/grails-app/controllers/org/gokb/CreateController.groovy

```diff
@@ -496,6 +496,7 @@ class CreateController {
               if (colMap.national_ranges) {
                 List<String> national_ranges = cols[colMap.national_ranges].split(',')
                 national_ranges.each { String value ->
+                  value = value.trim()
                   RefdataValue refdataValue = RefdataCategory.lookup(RCConstants.COUNTRY, value)
```

Comment:
include value.trim() directly as param in lookup -> RefdataCategory.lookup(RCConstants.COUNTRY, value.trim())